hazelcast/src/main/java/com/hazelcast/jet/impl/MasterJobContext.java

```diff
@@ -697,6 +697,10 @@ private Throwable getErrorFromResponses(String opName, Collection<Map.Entry<Memb
 
         if (!otherFailures.isEmpty()) {
             return (Throwable) otherFailures.get(0).getValue();
+        } else if (!isJobSuspendable(mc.jobConfig())) {
```

Comment:
should those be non-suspendable jobs, SQL jobs, SQL non-`CREATE JOB` jobs, non-suspendable SQL jobs?

Reply:
> non-suspendable jobs, SQL jobs, SQL non-CREATE JOB jobs, non-suspendable SQL jobs

They are synonyms to the "job, running with `KEY_JOB_IS_SUSPENDABLE = true` is `JobConfig`.